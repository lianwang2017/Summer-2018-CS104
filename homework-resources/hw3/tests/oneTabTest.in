Merrick Bautista	, CECS
David Kempe	, CSCI
Mark W. Redekopp	, EE