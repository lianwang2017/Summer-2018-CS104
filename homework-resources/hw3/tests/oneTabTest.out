CECS 
Merrick Bautista

CSCI 
David Kempe

EE 
Mark W. Redekopp

